templates/task/partials/task_list.html

```diff
@@ -1,5 +1,5 @@
-{# Closes the overlays if present using an out-of-band swap #}
-<div hx-swap-oob="outerHTML: .overlay-river"></div>
+{# Closes the task edit overlay if present using an out-of-band swap #}
+<div id="task-edit" hx-swap-oob="delete"></div>
```

Comment:
The previous method was removing all the overlays that are embedded in the page but hidden (oops! 🤦), so as soon as you had a task list, the river settings modals stopped working as they had been removed.

So, now to use this out-of-band swap method, you need to use an id for it, in combination with setting `overlay_id`, e.g. `{% block overlay_id %}task-edit{% endblock %}`.

Reply:
👌